.github/workflows/ci.yml

```diff
@@ -13,19 +13,19 @@ jobs:
           name: build artifacts
           path: build/
 
-  run-mocha:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-        node-version: [12.x, 14.x]
-    steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node-version }}
-      - run: yarn test
+  # run-mocha:
```

Comment:
why is this commented out? id rather us keep this with a dummy test, or delete wholesale

Reply:
@eligrey Let's just remove for now

Reply:
yeah removing this 👍